src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -87,16 +122,26 @@ const SearchResult = ({ form, showFindFormsResultsLinkToFormDetailPages }) => {
       ) : null}
 
       <dd className="vads-u-margin-bottom--1">
-        <a href={url} rel="noreferrer noopener" {...linkProps}>
+        <a
+          href={url}
+          rel="noreferrer noopener"
+          onClick={() =>
+            recordGAEvent(`Download VA form ${id} ${pdfLabel}`, url, 'pdf')
+          }
+          {...linkProps}
+        >
           Download VA form {id} {pdfLabel}
         </a>
       </dd>
 
-      {formToolUrl ? (
+      {showFindFormsResultsLinkToFormDetailPages && formToolUrl ? (
```

Comment:
Is this `showFindFormsResultsLinkToFormDetailPages` directly related to this work? No problem if not, just wondering

Reply:
Yes, I have to add the feature flag in for _some_ existing code which is suppose to be reviewed by VSP collab. I added the code before I was aware of the process yah know?